### ISSUE TITLE
tms endpoint - can multiple URIs be provided?

### ISSUE THREAD

**Issue by mojodna** [hotosm/oam-catalog#61]
We're passing TileJSON around the OAM server API, so we theoretically have multiple entries for `tiles`.  It'd be nice to not lose them if possible.  In the meantime, I'm grabbing the first and sending that as `uri`.

(Note: this is more relevant for Mapbox / other TMSes with host aliasing)

**Comment by anandthakker** [comment]
Good point @mojodna.  I haven't spent much time in here myself, but it looks to me like the underlying schema & controller are [setup to handle](https://github.com/hotosm/oam-catalog/blob/develop/controllers/meta.js#L184-L199) multiple tms uri's, but `POST /tms` only takes one.  

Should be easy enough to allow an array for the `uri` parameter.  @scisco or @jflasher :+1: or :-1: ?

**Comment by kamicut** [comment]
@jflasher @scisco I'm about to work on this, any red flags for just substituting the tms' single value with an array?

**Comment by jflasher** [comment]
@kamicut @scisco implemented this I believe, I don't know that I've looked into it previously.
